### PR TITLE
Atdding test cases for file sink and custom new.line.charater

### PR DESCRIPTION
**Pull request by Yasara123**

### CODE REVIEW

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java [hunk 1]
```diff
@@ -632,4 +634,194 @@ public String getTopic() {
         InMemoryBroker.unsubscribe(subscriberWSO2);
         InMemoryBroker.unsubscribe(subscriberIBM);
     }
+
+    @Test
+    public void testTextSinkMapperNewLineCharacterWrong() throws InterruptedException {
+        log.info("Test with custom new lie charater.");
```
Comment (dilini-muthumala) [inline component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java]: Please fix the typo :-) 'lie' should be 'line'

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java [hunk 2]
```diff
@@ -632,4 +634,194 @@ public String getTopic() {
...
+                "select * " +
+                "insert into BarStream; ";
+
+        SiddhiManager siddhiManager = new SiddhiManager();
+        siddhiManager.setExtension("sink:inMemory", InMemorySink.class);
+        SiddhiAppRuntime siddhiAppRuntime = siddhiManager.createSiddhiAppRuntime(streams + query);
+        InputHandler stockStream = siddhiAppRuntime.getInputHandler("FooStream");
+
+        siddhiAppRuntime.start();
+        ArrayList<org.wso2.siddhi.core.event.Event> arrayList = new ArrayList<>(100);
```
Comment (tishan89) [inline component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java]: Declaration should be a List

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java [hunk 3]
```diff
@@ -632,4 +634,194 @@ public String getTopic() {
...
+
+        siddhiAppRuntime.start();
+
+        stockStream.send(new Object[]{"WSO2", 55.6f, 100L});
+        stockStream.send(new Object[]{"IBM", 57.678f, 100L});
+        stockStream.send(new Object[]{"GOOGLE", 50f, 100L});
+        stockStream.send(new Object[]{"REDHAT", 50f, 100L});
+        Thread.sleep(100);
+
+        ArrayList<String> symbolNames = new ArrayList<>();
```
Comment (tishan89) [inline component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java]: Declaration should be a List

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java [hunk 4]
```diff
@@ -632,4 +634,194 @@ public String getTopic() {
...
+                "select * " +
+                "insert into BarStream; ";
+
+        SiddhiManager siddhiManager = new SiddhiManager();
+        SiddhiAppRuntime siddhiAppRuntime = siddhiManager.createSiddhiAppRuntime(streams + query);
+        InputHandler stockStream = siddhiAppRuntime.getInputHandler("FooStream");
+
+        siddhiAppRuntime.start();
+
+        ArrayList<org.wso2.siddhi.core.event.Event> arrayList = new ArrayList<>(100);
```
Comment (tishan89) [inline component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java]: Declaration should be a List

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java [hunk 5]
```diff
@@ -632,4 +634,194 @@ public String getTopic() {
...
+        for (int j = 0; j < 5; j++) {
+            arrayList.add(new org.wso2.siddhi.core.event
+                    .Event(System.currentTimeMillis(), new Object[]{"WSO2", 55.6f, 10}));
+            arrayList.add(new org.wso2.siddhi.core.event
+                    .Event(System.currentTimeMillis(), new Object[]{"IBM", 75.6f, 10}));
+        }
+        stockStream.send(arrayList.toArray(new org.wso2.siddhi.core.event.Event[10]));
+        Thread.sleep(100);
+
+        ArrayList<String> symbolNames = new ArrayList<>();
```
Comment (tishan89) [inline component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java]: Declaration should be a List

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextDefaultSinkMapperTestCase.java [hunk 6]
```diff
@@ -469,4 +534,170 @@ public String getTopic() {
...
+                "select * " +
+                "insert into BarStream; ";
+
+        SiddhiManager siddhiManager = new SiddhiManager();
+        SiddhiAppRuntime siddhiAppRuntime = siddhiManager.createSiddhiAppRuntime(streams + query);
+        InputHandler stockStream = siddhiAppRuntime.getInputHandler("FooStream");
+
+        siddhiAppRuntime.start();
+
+        ArrayList<org.wso2.siddhi.core.event.Event> arrayList = new ArrayList<>(100);
```
Comment (tishan89) [inline component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextDefaultSinkMapperTestCase.java]: Declaration should be a List

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextDefaultSinkMapperTestCase.java [hunk 7]
```diff
@@ -469,4 +534,170 @@ public String getTopic() {
...
+        for (int j = 0; j < 5; j++) {
+            arrayList.add(new org.wso2.siddhi.core.event
+                    .Event(System.currentTimeMillis(), new Object[]{"WSO2", 55.6f, 10}));
+            arrayList.add(new org.wso2.siddhi.core.event
+                    .Event(System.currentTimeMillis(), new Object[]{"IBM", 75.6f, 10}));
+        }
+        stockStream.send(arrayList.toArray(new org.wso2.siddhi.core.event.Event[10]));
+        Thread.sleep(100);
+
+        ArrayList<String> symbolNames = new ArrayList<>();
```
Comment (tishan89) [inline component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextDefaultSinkMapperTestCase.java]: Declaration should be a List

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextDefaultSinkMapperTestCase.java [hunk 8]
```diff
@@ -469,4 +534,170 @@ public String getTopic() {
...
+
+        siddhiAppRuntime.start();
+
+        stockStream.send(new Object[]{"WSO2", 55.6f, 100L});
+        stockStream.send(new Object[]{"IBM", 57.678f, 100L});
+        stockStream.send(new Object[]{"GOOGLE", 50f, 100L});
+        stockStream.send(new Object[]{"REDHAT", 50f, 100L});
+        Thread.sleep(100);
+
+        ArrayList<String> symbolNames = new ArrayList<>();
```
Comment (tishan89) [inline component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextDefaultSinkMapperTestCase.java]: Declaration should be a List

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java (hunk 2 above)
Comment (Yasara123) [reply component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java]: Fixed in cdb7b86019f370bcb90c545043b2ad205e8e98df

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java (hunk 3 above)
Comment (Yasara123) [reply component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java]: Fixed in cdb7b86019f370bcb90c545043b2ad205e8e98df

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java (hunk 5 above)
Comment (Yasara123) [reply component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java]: Fixed in cdb7b86019f370bcb90c545043b2ad205e8e98df

File: component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextDefaultSinkMapperTestCase.java (hunk 7 above)
Comment (Yasara123) [reply component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextDefaultSinkMapperTestCase.java]: Fixed in cdb7b86019f370bcb90c545043b2ad205e8e98df

**Pull request merged**